packages/console/src/components/AuditLogTable/components/EventName/index.tsx

```diff
@@ -0,0 +1,43 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import { logEventTitle } from '@/consts/logs';
+import Failed from '@/icons/Failed';
+import Success from '@/icons/Success';
+
+import * as styles from './index.module.scss';
+
+type Props = {
+  type: string;
+  isSuccess: boolean;
+  to?: string;
+};
+
+const EventName = ({ type, isSuccess, to }: Props) => {
+  const title = logEventTitle[type] ?? type;
+
+  return (
+    <div className={styles.eventName}>
+      <div className={classNames(styles.icon, isSuccess ? styles.success : styles.fail)}>
+        {isSuccess ? <Success /> : <Failed />}
+      </div>
+      <div>
+        {to && (
+          <Link
+            className={styles.title}
+            to={to}
+            onClick={(event) => {
+              event.stopPropagation();
+            }}
```

Comment:
Should we disable the onclick inside the `Link` component? So that once we pass in the `to` prop, the onClick will be automatically disabled